.clang-format

```diff
@@ -67,21 +64,11 @@ SortIncludes : true
 SortUsingDeclarations: true
 SpaceAfterTemplateKeyword: false
 SpaceBeforeAssignmentOperators: true
-
-#Unsupported in VS2017-included ClangFormat 6.0 we'll see about VS2019's ClangFormat 7.0
-#SpaceBeforeCpp11BracedList: false
-
-#Unsupported in VS2017-included ClangFormat 6.0 we'll see about VS2019's ClangFormat 7.0
-#SpaceBeforeCtorInitializerColon: true
-
-#Unsupported in VS2017-included ClangFormat 6.0 we'll see about VS2019's ClangFormat 7.0
-#SpaceBeforeInheritanceColon: true
-
+SpaceBeforeCpp11BracedList: false
```

Comment:
Would be nice to know what exactly this means. Maybe a reference link would help.

In particular, does this setting describe existing practice?

Reply:
Ref: https://clang.llvm.org/docs/ClangFormatStyleOptions.html
There are no anchor links, sorry. :(
You'll have to Ctrl+F search for the command.

> If true, a space will be inserted before a C++11 braced list used to initialize an object (after the preceding identifier or type).
```
    true:                                  false:
    Foo foo { bar };               vs.     Foo foo{ bar };
    Foo {};                                Foo{};
    vector<int> { 1, 2, 3 };               vector<int>{ 1, 2, 3 };
    new int[3] { 1, 2, 3 };                new int[3]{ 1, 2, 3 };
```

Reply:
The false option does.